Warzone/GameEngine.cpp

```diff
@@ -217,6 +217,10 @@ void StartUpPhase::startupPhase()
 	this->distrubuiteTerritories();
 	this->setReinforcements();
 	this->createOrderOfPlay();
+
+	for (int i = 0; i < this->getPlayers().size(); i++) {
```

Comment:
@chaim-f I don't why for some reason it is only giving 1 territory for each player (see output)
```
********** GAME START **********
Reading map directory...
> map 1: maps/canada.map
> map 2: maps/europe.map
done...
Loading maps...
done...
Storing map...
*** Extracting continents from maps/canada.map
*** Extracting countries/territories from maps/canada.map
*** Extracting countries/territories with borders from maps/canada.map
*** Extracting continents from maps/europe.map
*** Extracting countries/territories from maps/europe.map
*** Extracting countries/territories with borders from maps/europe.map
done...
Validating maps...
> validating maps/canada.map
Graph is connected!
> validating maps/europe.map
Graph is connected!
done...

Please choose a map...
Enter number 1 to choose maps/canada.map
Enter number 2 to choose maps/europe.map
2
maps/europe.map was chosen

Enter the number players. Must be a value between 2-5
3
Creating 3 players
------------------------------------------------------

Creating: Jerry

Creating: Robin

Creating: Batman

All Territories ranadomly allowcated
Filled each players reinforcement pool with 35 armies each
Territory(10, 9, Poland, Continent(2, East_Europe, 4))
Territory(11, 10, Czech_Rep, Continent(2, East_Europe, 4))
Territory(24, 21, Luxembourg, Continent(4, West_Europe, 3))
```